Data/Unicode/Properties/CombiningClass.hs

```diff
@@ -877,12 +876,14 @@ getCombiningClass _ = 0
 
 {-# INLINE isCombining #-}
 isCombining :: Char -> Bool
-isCombining c | (ord c) < 768 || (ord c) > 125258 = False
-isCombining c = combiningBitmap (ord c)
+isCombining c =
+    let i = ord c
+    in if i < 768 || i > 125258
+       then False
+       else combiningBitmap i
```

Comment:
Is there a reason to prefer this to `i >= 768 && i <= 125258 && combiningBitmap i`?

Reply:
I don't prefer anyone in particular.